contracts/schemes/TransitionScheme.sol

```diff
@@ -0,0 +1,97 @@
+pragma solidity 0.5.17;
+
+import "../controller/Controller.sol";
+
+/**
+ * @title A scheme for transitioning the DAO's assets to a new one.
+ */
+
+contract TransitionScheme {
+
+    event OwnershipTransferred(Avatar indexed _avatar, address indexed _newAvatar, address indexed _asset);
+
+    Avatar public avatar;
+    address payable public newAvatar;
+    address[] public externalTokens;
+    address[] public assetAddresses;
+    bytes4[] public selectors;
+
+    /**
+     * @dev initialize
+     * @param _avatar the avatar to migrate from
+     * @param _newAvatar the avatar to migrate to
+     * @param _externalTokens external tokens to allow transfer to the new avatar
+     * @param _assetAddresses the assets to transfer
+     * @param _selectors the functions to call to to transfer the assets
+     */
+    function initialize(
+        Avatar _avatar,
+        address payable _newAvatar,
+        address[] calldata _externalTokens,
+        address[] calldata _assetAddresses,
```

Comment:
how about cap the number of asserts with a number which is applicable ?

Reply:
Why? The number of assets is determined in the initialized, so the DAO knows how many and which assets will be transferred before adding the scheme.

Reply:
I mean in terms of gas usage ..while iterating over the assets.

Reply:
I think we can do enough assets at a time to make it unnecessary, but if you want we can add it. I could add a test to see how many we can do at a time but it'd be just a rough estimation as different assets will probably cost differently.

Reply:
test with normal transfer ownership

Reply:
ok I added a test transferring 101 assets.